### ISSUE TITLE
Readme or guidance on running CloneTrack

### ISSUE THREAD

**Issue by ksinghal28** [zsethna/CloneTrack#2]
Hello, thank you for developing the CloneTrack tool and making it available. I came across it in the "Personalized RNA neoantigen vaccines stimulate T cells in pancreatic cancer" publication, and would like to implement it on my dataset. 

I was curious if you have a Readme or some guidance available on how one can run the tool, the inputs format, which scripts to use and what order to use them in, etc? 

Thanks again,
Kartik

**Comment by biounix** [comment]
+1

**Comment by LutherScript** [comment]
+1

**Comment by LutherScript** [comment]
> +1

have you tested this method on your data?

**Comment by biounix** [comment]
> > +1
> 
> have you tested this method on your data?

No, mostly due to the lack of documentation. I ended up using an alternative R package instead: https://github.com/davidcoffey/LymphoSeq, which includes a function also called cloneTrack. Although it lacks a supporting publication like the CloneTrack in this repo, it is well-documented and was sufficient for my needs.
